### PR TITLE
Update HttpCache.php

### PR DESCRIPTION
**Pull request by reenigneEsrever92**
simplification...
I guess the original Author was trying to pursue something like a martingale (doubling) strategy (or in this case the increment over a constant factor) but well ... he did not. Instead a constant wait time was introduced, which means the wait is simply executed a 100 times over (in the worst case). And so I do not see a need for "5000000" and "+= 50000" to remain here.

| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: src/Symfony/Component/HttpKernel/HttpCache/HttpCache.php [hunk 1]
```diff
@@ -5,10 +5,6 @@
  *
  * (c) Fabien Potencier <fabien@symfony.com>
  *
- * This code is partially based on the Rack-Cache library by Ryan Tomayko,
- * which is released under the MIT license.
- * (based on commit 02d2b48d75bcb63cf1c0c7149c077ad256542801)
- *
```
Comment (fabpot) [inline src/Symfony/Component/HttpKernel/HttpCache/HttpCache.php]: This must be kept.

File: src/Symfony/Component/HttpKernel/HttpCache/HttpCache.php [hunk 2]
```diff
@@ -36,7 +32,7 @@ class HttpCache implements HttpKernelInterface, TerminableInterface
     private $traces = array();
 
     /**
-     * The available options are:
+     * The available options are:.
```
Comment (fabpot) [inline src/Symfony/Component/HttpKernel/HttpCache/HttpCache.php]: Dot must be removed

**Review by fabpot — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/HttpKernel/HttpCache/HttpCache.php (hunk 2 above)
Comment (keradus) [reply src/Symfony/Component/HttpKernel/HttpCache/HttpCache.php]: Sf follows PHPDoc here:
https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md#51-summary

first statement in docblock here is actually a `description`, while PHPDoc ask us to have `summary` first, that's why it got full stop added by PHP CS Fixer.

File: src/Symfony/Component/HttpKernel/HttpCache/HttpCache.php (hunk 1 above)
Comment (fabpot) [reply src/Symfony/Component/HttpKernel/HttpCache/HttpCache.php]: Can you revert this removal?

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
@jozzan Don't spend time on the headers :) Can you change the target to 3.4 instead of master? I think we are then go to merge. Thank you.

**Comment by stof** [comment]
Merging you master branch into 3.4 is a total no-go. We don't want to merge the 4.0 history into 3.4.

**Comment by stof** [comment]
you should instead rewrite the history of your branch so that it contains only your new commits on top of the 3.4 branch (and this is also why you should always use a feature branch rather than committing in `master` directly, as having your master branch out of sync with Symfony's one will become a pain for your fork)

**Pull request closed without merging**